Yoast/i18n-module
# If the request on Glotpress is wrong block the execution of the code to avoid errors

## Summary

This PR can be summarized in the following changelog entry:

This PR contains the contribution from @Mte90. This adds a check for a 404 response on the requests to Glotpress for translations.

## Relevant technical choices:
- If the request on Glotpress is wrong block the execution of the code to avoid errors

## Test instructions

This PR can be tested by following these steps:
1. Copy the changes to the i18n module in your wordpress-seo installation.
2. Change your Wordpress installation to a another language than english.
3. See if every translation still works.
4. Break the translation requests and check if it is correct.

Fixes #11
